tests/test_lime_survey.py

```diff
@@ -0,0 +1,87 @@
+"""Test functions related to Survey class"""
+import unittest
+
+import pandas as pd
+from pandas._testing import assert_frame_equal
+
+from n2survey.lime import LimeSurvey, read_lime_questionnaire_structure
+
+
+class TestLimeSurveyInitialisation(unittest.TestCase):
+    """Test LimeSurvey class initialisation"""
+
+    def assert_df_equal(self, df1, df2, msg):
+        """DataFrame equality test from pandas"""
+        try:
+            assert_frame_equal(df1, df2)
+        except AssertionError:
+            raise self.failureException(msg)
+
+    def setUp(self):
```

Comment:
I guess you can also add "structure_file" here since you are using it in all tests